### DISCUSSION TITLE
Installation Help

### DISCUSSION THREAD

**Discussion by UK-17** [h2oai/wave#1338] [Q&A]
How do I install pre-release versions?

I cannot seem to use pip to install the pre-release. The tar.gz file also doesn't have setup.py. How can I install the pre-release in my python virtual environment ?

**Comment by mturoci** [comment]
Good question @UK-17!

All you need to do is to download the `.whl` file based on the system you use and then run `pip install <downloaded>.whl` - this is an alternative to `pip install h2o-wave`.

If interested, more info can be found [here](https://github.com/h2oai/wave/blob/master/website/docs/installation.mdx#early-adopters).
